src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/providers/ProjectApi.java

```diff
@@ -0,0 +1,65 @@
+package network.radicle.jetbrains.radiclejetbrainsplugin.providers;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import network.radicle.jetbrains.radiclejetbrainsplugin.models.RadProject;
+import network.radicle.jetbrains.radiclejetbrainsplugin.models.SeedNode;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.util.EntityUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ProjectApi {
+    private static final Logger logger = LoggerFactory.getLogger(ProjectApi.class);
+    private static final int PER_PAGE = 10;
+    private HttpClient client;
+
+    public ProjectApi() {
+        this.client = HttpClientBuilder.create().build();
+    }
+
+    public ProjectApi(HttpClient client) {
+        this.client = client;
+    }
+
+    public  List<RadProject> fetchRadProjects(SeedNode selectedNode, int page) {
+        var url = "https://" + selectedNode.host + ":" + selectedNode.port + "/v1/projects?per-page=" + PER_PAGE +
```

Comment:
I'm not sure if this is correct.
The seed node host / port is assumed to be for the rad protocol, not the radicle server's http api.